File: src/my_project/easy_problems/from251to300/length_of_last_word.py
from typing import List 

class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        
        s = s.strip()
        
        temp = 0
        for i in range(len(s)):
            if s[i] == " ":
                temp = 0
                continue
            temp += 1

        return temp
        
        
        
solution = Solution()

print(solution.lengthOfLastWord("   fly me   to   the moon       "))

File: my_project_typescript/easy_problems_typescript/from151to200/lengthOfLastWord.ts
function lengthOfLastWord(s: string): number {

    s = s.trim()
    let temp: number = 0
    for (let i  = 0; i < s.length; i++){
        if (s[i] ===  " "){
            temp = 0
            continue
        }
        temp += 1
    }

    return temp 

};